tests/integration/sriov_test.py

```diff
@@ -95,7 +95,10 @@ def sriov_interface(disable_sriov):
         Interface.NAME: pf_name,
         Interface.STATE: InterfaceState.UP,
         Ethernet.CONFIG_SUBTREE: {
-            Ethernet.SRIOV_SUBTREE: {Ethernet.SRIOV.TOTAL_VFS: 2},
+            Ethernet.SRIOV_SUBTREE: {
+                Ethernet.SRIOV.DRIVERS_AUTOPROBE: True,
+                Ethernet.SRIOV.TOTAL_VFS: 2,
```

Comment:
I honestly think that we do not need to patch other tests to add `Ethernet.SRIOV.DRIVERS_AUTOPROBE: True` property if it is not mandatory to specify (at least I can not tell that it is mandatory from the setting that you specified in `rust/src/sriov.rs` ).

Reply:
Actually, after setting it to `False` once, it will not reset to `True` implicitly. I haven't looked at the NetworkManager code base but if it is only changing the value when nmstate explicitly asks for it, then `sriov_drivers_autoprobe` will remain set to `0` in sysfs.

I am now wondering if nmstate should force `autoprobe-drivers=1` in NetworkManager if the setting is unset in nmstate.

Reply:
I have removed `Ethernet.SRIOV.DRIVERS_AUTOPROBE: True` as you requested and modified the NetworkManager settings code path to set autoprobe-drivers=true if it is unspecified by nmstate.

Reply:
Yes, I think that this way will align better with the behavior in NM.

Reply:
```
       sriov.autoprobe-drivers
           Whether to autoprobe virtual functions by a compatible driver.

           If set to NM_TERNARY_TRUE (1), the kernel will try to bind VFs to a compatible driver and if this succeeds a new network interface will be instantiated for each VF.

           If set to NM_TERNARY_FALSE (0), VFs will not be claimed and no network interfaces will be created for them.

           When set to NM_TERNARY_DEFAULT (-1), the global default is used; in case the global default is unspecified it is assumed to be NM_TERNARY_TRUE (1).

           Format: NMTernary (int32)

```